ci/coverage_publish.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Do not ever set -x here, it is a security hazard as it will place the credentials below in the
+# Travis logs.
+set -e
+
+if [ "${TRAVIS}" != "true" ]; then 
+  exit 0
+fi
+
+# available for master builds and PRs from originating repository (not forks)
+if [ "${TRAVIS_SECURE_ENV_VARS}" == "true" ]; then
+  mkdir -p ~/.aws
+  echo "[profile coverage]" >> ~/.aws/config
+  echo "aws_access_key_id=${COVERAGE_AWS_ACCESS_KEY_ID}" >> ~/.aws/config
+  echo "aws_secret_access_key=${COVERAGE_AWS_SECRET_ACCESS_KEY}" >> ~/.aws/config
+  echo "region=us-east-1" >> ~/.aws/config
```

Comment:
Do we want to cleanup the config when done since it contains creds, or do we have guarantees that Travis nukes it for us?